starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -988,7 +995,7 @@ List<Property> properties
               </sourceDirs>
             </configuration>
           </execution>
-@if (features.testFramework().isKotlinTest() || features.testFramework().isJunit()) {
+@if (features.testFramework().isKotlinTest() || features.testFramework().isKoTest() || features.testFramework().isJunit()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411